### PR TITLE
Add the example: node affinity

### PR DESCRIPTION
**Pull request by farmer-hutao**
Add the example: Run a pod on a node with an Intel or AMD CPU and in availability zone Z

### CODE REVIEW

File: contributors/design-proposals/scheduling/nodeaffinity.md [hunk 1]
```diff
@@ -144,12 +144,36 @@ Hopefully this won't cause too much confusion.
...
+          - key: cpu-arch
+            operator: In
+            values:
+            - intel
+            - amd
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 1
+        preference:
+          matchExpressions:
+          - key: zone
```
Comment (bsalamat) [inline contributors/design-proposals/scheduling/nodeaffinity.md]: zone is a well-known label in Kubernetes. Please use `failure-domain.kubernetes.io/zone`.

Comment (farmer-hutao) [reply contributors/design-proposals/scheduling/nodeaffinity.md]: ok, I'll do it immediately

File: contributors/design-proposals/scheduling/nodeaffinity.md [hunk 2]
```diff
@@ -144,12 +144,36 @@ Hopefully this won't cause too much confusion.
...
+          - key: kubernetes.io/arch
+            operator: In
+            values:
+            - intel
+            - amd64
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 1
+        preference:
+          matchExpressions:
+          - key: failure-domain.beta.kubernetes.io/zone
```
Comment (bsalamat) [inline contributors/design-proposals/scheduling/nodeaffinity.md]: The label is already promoted to GA. Please use `failure-domain.kubernetes.io/zone`

**Comment by bsalamat** [comment]
Please squash commits.

File: contributors/design-proposals/scheduling/nodeaffinity.md (hunk 2 above)
Comment (farmer-hutao) [reply contributors/design-proposals/scheduling/nodeaffinity.md]: Oh, I'm sorry. I'll update it and squash commits today.

**Comment by farmer-hutao** [comment]
I'll Reopen a pr to squash 3 commits

**Comment by farmer-hutao** [comment]
#3525

**Pull request closed without merging**